<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestProcess</class>
 <widget class="QDialog" name="TestProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Home_TestProcess_lbBK1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_TestProcess_lbBK2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Home_TestProcess_lbTestName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_TestProcess_lbTestSerial">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_TestProcess_lbTestType">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_TestProcess_lbSampleCode">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_TestProcess_lbSampleInfo">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Home_TestProcess_lbSampleRemark">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Home_TestProcess_btCancelTest">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
